common/changes/inateeg-1201popup-elit_2017-01-13-00-11.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Support Nav horizontal orentation rendering",
```

Comment:
This language needs to be in a better format; this goes into the public release notes, so please use this format:

Component: Full sentence explaining the new fix/change/feature.

Example:

Nav: Now adding the ability to render the Nav horizontally, if the `horizontal` attribute is provided.